### DISCUSSION TITLE
Compatibility with gpt4all-j

### DISCUSSION THREAD

**Discussion by rabidcopy** [LostRuins/koboldcpp#58] [General]
It appears over at [gpt4all](https://github.com/nomic-ai/gpt4all) they have moved towards gpt-j. As such they have released a [ggml](https://gpt4all.io/ggml-gpt4all-j.bin). They now direct users to use https://github.com/nomic-ai/gpt4all-chat for inference. Most of this seems to be based on ggerganov's ggml repo. One thing of note is their gptj.cpp isn't up to date with https://github.com/ggerganov/ggml/commit/0265f0813492602fec0e1159fe61de1bf0ccaf78. So first thought is, "huh, wonder if this will load fine on koboldcpp?", answer is, yep!

**Comment by LostRuins** [comment]
Yeap, KoboldCpp should handle all formats automatically as it infers the file type at load time. Let me know if a specific format is broken.
